src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -156,7 +156,7 @@ IntPtr overlapped
             );
 
             // older Windows systems:
-            [DllImport(@"kernel32.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
+            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
```

Comment:
> Can we revert this for now and track if/how to address in a separate bug?

Note that both Pinvoke's are incorrect i.e. kernel32!ReadFile and api-ms-win-core-file-l1-1-0!ReadFile since this library is built as AnyCPU while these methods are windows specific. I added exception for api-ms-win-core-file-l1-1-0!ReadFile, however unfortunately,  as the infrastructure stands as of now, I can't add exception to both entries. We currently only support one methodName exception (there is really no reason, other than in this special case, where we want to support multiple module name's for the same entry point) and honestly, I don't think we should add that support in the infrastructure either.

The right fix would be to separate the Desktop implementation and make this library windows specific (or remove windows Pinvokes from the x-plat implementation). A temporary workaround would be to disable the CodeAnalyzers for this project (however note that all code analyzers will be disabled this way). May be adding a separate rule set or property will work but I have not tried that yet.